InnerEye/ML/dataset/sample.py

```diff
@@ -132,14 +132,15 @@ class PatientDatasetSource(SampleBase):
     ground_truth_channels: List[PathOrString]
     mask_channel: Optional[PathOrString]
     metadata: PatientMetadata
+    allow_incomplete_labels: bool
```

Comment:
I don't see why this flag needs to be here? It would be more sensible to have ground_truth_channels as List[Optional[Path]], clearly indicating that some paths can be missing.

Reply:
Or add a field to the Sample class saying which ground_truth_channels are missing?